ObservableWebView/Toolbar/UrlSearchBar/UrlSearchBar.swift

```diff
@@ -39,11 +40,10 @@ struct UrlSearchBarTextField: View {
               showTextField = false
             }
           
-          TextField("Search or type URL", text: $text)
-            .textFieldStyle(.plain)
-            .onSubmit {
-              manager.load(text)
-            }
+          UrlSearchBarTextField(text: $text, isEditing: $isEditing) {
+            manager.load(text)
+          }
+          .textFieldStyle(.plain)
           
           if !text.isEmpty {
             Image(systemName: "xmark.circle.fill")
```

Comment:
May not be required anymore with selectAll on appear functionality. Take note while testing.